assets/js/components/wp-dashboard/WPDashboardPopularPages.js

```diff
@@ -0,0 +1,126 @@
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { MODULES_ANALYTICS, DATE_RANGE_OFFSET } from '../../modules/analytics/datastore/constants';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import PreviewBlock from '../PreviewBlock';
+import ReportError from '../ReportError';
+import { numFmt } from '../../util';
+import TableOverflowContainer from '../../components/TableOverflowContainer';
+import { getDataTableFromData } from '../../components/data-table';
+import ReportZero from '../../components/ReportZero';
+import { isZeroReport } from '../../modules/analytics/util/is-zero-report';
+const { useSelect } = Data;
+
+const WPDashboardPopularPages = () => {
+	const dateRangeDates = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
+		compare: true,
+		offsetDays: DATE_RANGE_OFFSET,
+	} ) );
+
+	const reportArgs = {
+		...dateRangeDates,
+		metrics: [
+			{
+				expression: 'ga:pageviews',
+				alias: 'Pageviews',
+			},
+		],
+		dimensions: [
+			'ga:pageTitle',
+			'ga:pagePath',
+		],
+		orderby: [
+			{
+				fieldName: 'ga:pageviews',
+				sortOrder: 'DESCENDING',
+			},
+		],
+		limit: 5,
+	};
+
+	const data = useSelect( ( select ) => select( MODULES_ANALYTICS ).getReport( reportArgs ) );
+	const error = useSelect( ( select ) => select( MODULES_ANALYTICS ).getErrorForSelector( 'getReport', [ reportArgs ] ) );
+	const loading = useSelect( ( select ) => ! select( MODULES_ANALYTICS ).hasFinishedResolution( 'getReport', [ reportArgs ] ) );
+
+	if ( loading ) {
+		return <PreviewBlock width="48%" height="92px" />;
+	}
+
+	if ( error ) {
+		return <ReportError moduleSlug="analytics" error={ error } />;
+	}
+
+	if ( isZeroReport( data ) ) {
+		return <ReportZero moduleSlug="analytics" />;
+	}
+
+	const headers = [
+		{
+			title: __( 'Title', 'google-site-kit' ),
+			tooltip: __( 'Page Title', 'google-site-kit' ),
+			primary: true,
+		},
+		{
+			title: __( 'Pageviews', 'google-site-kit' ),
+			tooltip: __( 'Pageviews', 'google-site-kit' ),
+		},
+	];
+	const links = [];
+	const dataMapped = data[ 0 ].data.rows.map( ( row, i ) => {
+		const [ title, url ] = row.dimensions;
+		links[ i ] = url;
+
+		return [
+			title,
+			numFmt( row.metrics[ 0 ].values[ 0 ], { style: 'decimal' } ),
+		];
+	} );
+	const options = {
+		hideHeader: false,
+		chartsEnabled: false,
+		links,
+		hideColumns: {
+			mobile: [ 2, 3 ],
+		},
```

Comment:
The `options` here are not correct, which causes the table to have a different appearance between the old and new (the new one is missing the URLs visually, and it's linking to the wrong location). See here:
![Screen Shot 2021-02-02 at 11 44 47 AM](https://user-images.githubusercontent.com/3531426/106653749-1342d900-654c-11eb-8bfd-0fc655aa0b9d.png)

I think you took them from the wrong component - they need to come from `WPAnalyticsDashboardWidgetTopPagesTable`, which is the legacy version of this.

Reply:
Good catch, thank you